tool/experimental/errmon/implementation/sentry/errmon.go

```diff
@@ -259,7 +265,6 @@ func EventToSentry(ev *errmontypes.Event) *sentry.Event {
 			Type:       "error",
 			Value:      ev.Error.Error(),
 			Module:     FuncNameToSentryModule(ev.Caller.Func().Name()),
-			ThreadID:   "goroutine",
```

Comment:
Apparently in some versions of `Sentry` this field is a number, and not a string.